prime-router/src/test/kotlin/history/azure/DeliveryFunctionTests.kt

```diff
@@ -432,30 +432,31 @@ class DeliveryFunctionTests : Logging {
         val action = Action()
         action.actionId = 550
         action.sendingOrg = organizationName
-        action.actionName = TaskAction.receive
+        action.actionName = TaskAction.send
         every { mockDeliveryFacade.fetchActionForReportId(any()) } returns action
         every { mockDeliveryFacade.fetchAction(any()) } returns null // not used for a UUID
-        every { mockDeliveryFacade.findDetailedDeliveryHistory(any(), any()) } returns returnBody
+        every { mockDeliveryFacade.findDetailedDeliveryHistory(any()) } returns returnBody
         every { mockDeliveryFacade.checkSenderAccessAuthorization(any(), any()) } returns true
         response = function.getDeliveryDetails(mockRequest, goodUuid)
         assertThat(response.status).isEqualTo(HttpStatus.OK)
         var responseBody: ExpectedDelivery = mapper.readValue(response.body.toString())
         assertThat(responseBody.deliveryId.toLong()).isEqualTo(returnBody.actionId)
         assertThat(responseBody.receivingOrg).isEqualTo(returnBody.receivingOrg)
 
-        // Good uuid, but not a 'receive' step report.
+        // Good uuid, but not a with `process` action step report.
```

Comment:
I think you need to remove the 'not a' in this sentence

Reply:
fixed